### ISSUE TITLE
Latest package on npmjs

### ISSUE THREAD

**Issue by jewelsonn** [wobsoriano/vagmi#49]
Hey, Can you build and release the latest build so that we could use that build?
Thank You.

**Comment by wobsoriano** [comment]
Yeah, I need to update some packages this weekend and check deprecated stuff before I release a new one.

**Comment by reslear** [comment]
the latest version is v1 has breaking changes and need migration from `ethers` to `viem`
